packages/core/docs/integrate/integration-guide.md

```diff
@@ -400,10 +395,14 @@ You need to create a couple of Vue components:
 |----------------------------------------------|-------------------------------------|-------------|
```

Comment:
`You need to create a couple of Vue components` - not each one in this table is a Vue component. Let's rephrase this